Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -276,51 +276,50 @@ public TestsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         /// Test Case for getting the questions by passing a particular category Id
         /// </summary>
         /// <returns></returns>
-        [Fact]
-        public async Task GetAllTestCategoryQuestionsByIdAsync()
-        {
-            var categoryList = new List<DomainModel.Models.Category.Category>();
-            var category1 = CreateCategory("category1");
-            await _categoryRepository.AddCategoryAsync(category1);
-            categoryList.Add(category1);
-            var category2 = CreateCategory("category2");
-            await _categoryRepository.AddCategoryAsync(category2);
-            categoryList.Add(category2);
-            var categoryAcList = Mapper.Map<List<DomainModel.Models.Category.Category>, List<CategoryAC>>(categoryList);
-            string userName = "asifkhan.ak95.ak@gmail.com";
-            //Configuring Application User
-            ApplicationUser user = new ApplicationUser() { Email = userName, UserName = userName };
-            await _userManager.CreateAsync(user);
-            var applicationUser = await _userManager.FindByEmailAsync(user.Email);
+        //[Fact]
+        //public async Task GetAllTestCategoryQuestionsByIdAsync()
+        //{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)